go.mod

```diff
@@ -156,7 +156,7 @@ replace (
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.2-0.20220218085826-7f949c0aa1d6
 
 	// TODO: remove when middleware will be implemented
-	github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.0.2-hooks
+	github.com/cosmos/ibc-go/v2 => github.com/yihuang/ibc-go/v2 v2.2.0-hooks
```

Comment:
change to crypto-org-chain org after tested.

Reply:
so the `v2.2.x` is a temp version in your local repo, the final version will be followed by the cosmos SDK release version?

Reply:
so the `v2.2.x` is a temp version in your local repo, the final version will be followed by the cosmos SDK release version?

Reply:
https://github.com/cosmos/ibc-go/tree/release/v2.2.x
ibc-go have created this release branch, just haven't released the v2.2.0 yet.